readthedocsext/theme/templates/errors/dashboard/403_csrf.html

```diff
@@ -0,0 +1,48 @@
+{% extends "errors/dashboard/base.html" %}
+
+{% load blocktrans trans from i18n %}
+
+{% block title %}
+  {% trans "Access restricted" %}
+{% endblock title %}
+
+{% block error_status %}
+  403
+{% endblock error_status %}
+{% block error_title %}
+  {{ block.super }}
+  {% trans "Access restricted" %}
+{% endblock error_title %}
+{% block error_subtitle_text %}
+{% endblock error_subtitle_text %}
+
+{% block error_icon_computer_name %}
+  fa-lock-keyhole
+{% endblock error_icon_computer_name %}
+{% block error_icon_mobile_name %}
+  fa-lock-keyhole
+{% endblock error_icon_mobile_name %}
+{% block error_icon_computer_style %}
+  --fa-primary-color: orange;
+{% endblock error_icon_computer_style %}
+{% block error_icon_mobile_style %}
+  --fa-primary-color: orange;
+{% endblock error_icon_mobile_style %}
```

Comment:
Instead of duplicating all this you can extend 403.html so the pages are kept in sync

```suggestion
{% extends "errors/dashboard/403.html" %}

{% load blocktrans trans from i18n %}
```